AboutPage.html

```diff
@@ -0,0 +1,123 @@
+  <link rel="stylesheet" href="/css/about.css">
+  <title>CapstoneFernando</title>
+</head>
+<body>
+  <div class="upVar">
+    <ul class="Uplist">
+      <a href=""><img src="/images/facebook.png" class="logFaceAndTwit" srcset=""></a>
+      <a href=""><img src="/images/twitter.png" class="logFaceAndTwit" srcset=""></a>
+      <a href="">English</a>
+      <a href="">My page</a>
+      <a href="">Logout</a>
+    </ul>
+  </div>
+  <header class="header">
+    <div>
+      <nav class="mobile-navbar">
+        <button><img src="./images/mobile_menu 1.png" alt="burgerMenu" onclick ="opneNave();" class="image-burger-menu"></button>
+      </nav>
+    </div>
+    
+    <div class="container-logo">
+      <a href="index.html"><img src="./images/LogoUrban.png" alt= My Logo class="logo"></a>
+    </div>
+      <nav class="navbar">
+        <ul class="nav-list"> 
+          <li>
+            <a href="AboutPage.html">About</a>
+          </li>
+          <li>
+            <a href="#Program">Program</a>
+          </li>
+          <li>
+            <a href="#Join">Join</a>
+          </li>
+          <li>
+          <li>
+            <a href="#Sponsor">Sponsor</a>
+          </li>
+          <li>
+            <a href="#News">News</a>
+          </li>
+            <a href="#" class="header_button" id="header_button">UC Campaigns</a>
+          </li>
+        </ul>
+      </header>
+
+  <main>
+    <section class="mobile-menu" id="mobileNave">
+      <ul class="nav-list2"> 
+        <img onclick="closeNave();" class="img-close" src="./images/Icon_X_.png" alt="">
+        <li class="mobile-link" onclick="closeNave();"><a href="AboutPage.html" >About</a></li>
+        <li class="mobile-link" onclick="closeNave();"><a href="#aboutme-container" >Program</a></li>
+        <li class="mobile-link" onclick="closeNave();"><a href="#contact">News</a></li>
+    </section>
+
+    <section class="home-about" id="home-about" >
+      <div class="containerHome-Page2">
+        <div>
+          <h1 class="about-title" id="about-title" >Urban Art Festival <br>Creativity in the Streets 2023</h1>
+        </div>
+        <div class="home-Description">
+          <p>
+            Welcome to the Urban Art Festival "Creativity in the Streets," where imagination takes to the streets and creativity knows no bounds. At the festival, we are dedicated to bringing urban art to life and creating a vibrant platform for artists, enthusiasts, and culture lover, Founded with a vision to transform the city into a canvas of artistic expression, the Urban Art Festival "Creativity in the Streets" was born. From its inception in [año de fundación], our festival has evolved into an annual celebration that redefines the way we experience art in urban spaces. Our journey has been marked by [menciona algunos hitos clave, por ejemplo: "awe-inspiring murals, dynamic live performances, and a growing community of artists"
+          </p>
+        </div>
+        <div>
```

Comment:
- [ ] I wanted to let you know that I noticed a problem, it looks like the font-family, sizes, and font-weight might need a little adjustment across your whole page. Remember that you need to use these font-sizes: Cocogoose, Lato.

   ### Your design:
   ![image](https://github.com/Fernando426/CapstoneN01/assets/107952724/5a852b54-4757-44ef-8e9a-c16d66eb777b)

   ### Behance design:
   ![image](https://github.com/Fernando426/CapstoneN01/assets/107952724/89f89b0c-698d-4809-9358-041b9d93baf4)